display_information/display-annotation/README.metadata.json

```diff
@@ -0,0 +1,28 @@
+{
+    "category": "Layers",
```

Comment:
little rusty here, but doesn't look like we have a category for "Layers": looking at the package where this samples is saved, this should be "Display information"

Reply:
You're right, I've missed on changing this. Amended